test/powershell/Language/Scripting/OrderedAttributeForHashTables.Tests.ps1

```diff
@@ -0,0 +1,70 @@
+﻿Describe 'Test for cmdlet to support Ordered Attribute on hash literal nodes' -Tags "CI" {
+    BeforeAll {
+        If (-not $IsCoreCLR) {
+            Get-WmiObject -Query "select * from win32_environment where name='TestWmiInstance'"  | Remove-WmiObject 
+        }
+    }
+    AfterAll {
+        If (-not $IsCoreCLR) {
+            Get-WmiObject -Query "select * from win32_environment where name='TestWmiInstance'"  | Remove-WmiObject 
+        }
+    }
+
+    It 'New-Object - Property Parameter Must take IDictionary' {
+	   $a = new-object psobject -property ([ordered]@{one=1;two=2})
+
+       $a | Should Not Be $null
+       $a.one | Should Be 1
```

Comment:
fixed al indentation in this request

---
In reply to: [77377165](https://github.com/Powershell/powershell/pull/2118#discussion_r77377165) [](ancestors = 77377165)